src/test/scala/mesosphere/marathon/core/instance/update/InstanceUpdateOpResolverTest.scala

```diff
@@ -189,29 +199,25 @@ class InstanceUpdateOpResolverTest extends UnitTest with Inside {
 
     // Mesos 1.1 task statuses specs. See https://mesosphere.atlassian.net/browse/DCOS-9941
 
-    "Processing a TASK_FAILED update for running task" in new Fixture {
-      val builder = TestInstanceBuilder.newBuilder(appId)
-      val instance = builder.addTaskRunning().getInstance()
-      val update = TaskStatusUpdateTestHelper.failed(instance)
-      val stateChange = updateOpResolver.resolve(Some(instance), update.operation)
+    "expunge after TASK_FAILED update for running decommissioned" in new Fixture {
+      val update = TaskStatusUpdateTestHelper.failed(existingDecommissionedInstance)
+      val stateChange = updateOpResolver.resolve(Some(existingDecommissionedInstance), update.operation)
 
       Then("result in an expunge")
       stateChange shouldBe a[InstanceUpdateEffect.Expunge]
     }
 
-    "Processing a TASK_GONE update for a running task" in new Fixture {
-      val builder = TestInstanceBuilder.newBuilder(appId)
-      val instance = builder.addTaskRunning().getInstance()
-      val update = TaskStatusUpdateTestHelper.gone(instance)
-      val stateChange = updateOpResolver.resolve(Some(instance), update.operation)
+    "expunge after TASK_GONE for a running decommissioned instance" in new Fixture {
```

Comment:
Do we have tests covering the case of simply updating an instance with a `TASK_GONE` update when the goal is `Running`?

Reply:
I have added that one